libs/styles/consonant-play-button.css

```
@@ -0,0 +1,115 @@
+
+ .modal-img-link {
+  position: relative;
+  display: inline-block;
+  height: 100%;
+  z-index: 4;
+}
+
+.modal-img-link .consonant-play-btn {
+  position: absolute;
+  height: 77px;
+  width: 77px;
+  left: calc(50% - 38.5px);
+  top: calc(50% - 38.5px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: var(--link-color-dark);
+  box-sizing: border-box;
+  border: 2.5px solid var(--color-white);
+  border-radius: 50%;
+  outline: none;
+}
+
+.modal-img-link .consonant-play-btn.btn-small {
+  height: 44px;
+  width: 44px;
+  left: calc(50% - 22px);
+  top: calc(50% - 22px);
+  border: 2px solid var(--color-white);
+}
+
+.modal-img-link .consonant-play-btn:hover {
+  background-color: var(--color-accent-down);
+}
+
+.modal-img-link .consonant-play-btn:focus,
+.modal-img-link .consonant-play-btn:focus-visible {
+  outline: 2.5px solid var(--link-color-dark);
+  outline-offset: 0;
+  background-color: var(--color-accent-down);
+}
+
+.modal-img-link .consonant-play-btn.btn-small:focus,
+.modal-img-link .consonant-play-btn.btn-small:focus-visible {
+  outline: 2px solid var(--link-color-dark);
```

```suggestion
  outline-width: 2px;
```
We don't need to duplicate declarations, only whats being changed.

---

@ryanmparrish I have made the suggested changes.